src/containers/Layout/Layout.tsx

```diff
@@ -27,8 +28,8 @@ const Layout = () => {
   const navigate = useNavigate();
   const { t } = useTranslation('common');
   const { config, accessModel } = useConfigStore(({ config, accessModel }) => ({ config, accessModel }), shallow);
-  const { menu, assets, siteName, description, integrations, styling, features } = config;
-  const cleengId = integrations?.cleeng?.id;
+  const { menu, assets, siteName, description, styling, features } = config;
+  const { clientId } = useClientIntegration();
```

Comment:
Maybe a name like `authServiceId` or `authProviderId` would make this more intuitive?

Reply:
Hmm, not sure if `authServiceId` is semantic enough... Maybe integrationId.. not sure what's best :D

Reply:
I think the integration that we care about here will always at least be an auth service (it may be an auth+subscription service, but it always needs auth) so I would go more specific than generic, especially because it's used for login based checks here. Not a blocker if we can't agree on a more readable name for now, maybe it will become more obvious in the way we talk about it as the project progresses.